models/articles.model.js

```diff
@@ -16,4 +16,17 @@ function selectArticle(article_id) {
 
 }
 
-module.exports = {selectArticle}
\ No newline at end of file
+function allArticlesData() {
+    return db.query(
+        `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.article_img_url, SUM(comments.votes) AS votes, COUNT(comments.body) AS comment_count
+        FROM articles 
```

Comment:
The SQL syntax to have the count return a number data type is `COUNT(comments.body)::INT` which will save you having to map over the result. 
I'm not sure the votes should be coming from the comments, the articles have their own `votes` property that just needs to be selected, there is no need to _SUM(comments.votes)_